src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Formatting/RazorFormattingPass.cs

```diff
@@ -355,9 +354,9 @@ static bool IsSingleLineDirective(SyntaxNode node, [NotNullWhen(true)] out Synta
         }
     }
 
-    private static void FormatWhitespaceBetweenDirectiveAndBrace(SyntaxNode node, RazorDirectiveSyntax directive, List<TextEdit> edits, RazorSourceDocument source, FormattingContext context)
+    private void FormatWhitespaceBetweenDirectiveAndBrace(SyntaxNode node, RazorDirectiveSyntax directive, List<TextEdit> edits, RazorSourceDocument source, FormattingContext context, bool forceNewLine)
```

Comment:
This looks like it can still be static

Reply:
Ahh, yes, thanks, good catch. Previously I had this method checking the option, rather than a parameter.